walletlibrarydemo/src/main/java/com/microsoft/walletlibrarydemo/feature/issuance/presentationlogic/SampleViewModel.kt

```diff
@@ -0,0 +1,50 @@
+package com.microsoft.walletlibrarydemo.feature.issuance.presentationlogic
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.net.Uri
+import androidx.lifecycle.ViewModel
+import com.microsoft.walletlibrary.VerifiedIdClientBuilder
+import com.microsoft.walletlibrary.requests.VerifiedIdRequest
+import com.microsoft.walletlibrary.requests.input.VerifiedIdRequestURL
+import com.microsoft.walletlibrary.verifiedid.VerifiedId
+import com.microsoft.walletlibrarydemo.db.VerifiedIdDatabase
+
+class SampleViewModel(@SuppressLint("StaticFieldLeak") val context: Context): ViewModel() {
+    var verifiedIdRequestResult: Result<VerifiedIdRequest<*>>? = null
```

Comment:
nit: why not just unwrap it and store it as VerifiedIdRequest?

Reply:
Because error handling was done in Fragment, this had to be Result. Moved error handling to view model and unwrapped the results here.